neuralmonkey/evaluators/beer.py

```diff
@@ -0,0 +1,66 @@
+# tests: lint, mypy
+
+import tempfile
+import subprocess
+# pylint: disable=unused-import
+from typing import List
+
+from neuralmonkey.logging import log
+
+
+# pylint: disable=too-few-public-methods
+
+class BeerWrapper(object):
+    """Wrapper for BEER scorer"""
+    # https://github.com/stanojevic/beer
+
+    def __init__(self, wrapper, name="BEER", encoding="utf-8"):
+        self.wrapper = wrapper
+        self.encoding = encoding
+        self.name = name
+
+    def serialize_to_bytes(self, sentences):
+        # type: (List[List[str]]) -> bytes
+        joined = [" ".join(r) for r in sentences]
+        string = "\n".join(joined) + "\n"
+        return string.encode(self.encoding)
+
+    def __call__(self, decoded, references):
+        # type: (List[List[str]], List[List[str]]) -> float
```

Comment:
This is an old annotation style, since we are using Python 3.5 we prefer writing the type annotations right after the arguments names as described [PEP 484](https://www.python.org/dev/peps/pep-0484/). Don't worry about it, we will unify the style (#329).